modules/overrides/overrides.go

```diff
@@ -6,7 +6,8 @@ import (
 	"io"
 	"time"
 
-	"github.com/cortexproject/cortex/pkg/util/runtimeconfig"
+	"github.com/cortexproject/cortex/pkg/util/log"
```

Comment:
does Tempo have it's own global logger? :)

Reply:
I tried finding one. AFAIK they use this global logger in other places as well.